portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsets.testcase

```diff
@@ -318,14 +318,88 @@ definition {
 	}
 
 	@description = "Verify that a Field with the Autocomplete Property persists after being added to a Form via Fieldset"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test PreserveTheAutoCompleteProperty {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133464 PreserveTheAutoCompleteProperty pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddElementSet();
+
+		Form.editName(formName = "Element Set 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Numeric",
+			formPageNumber = "0");
+
+		FormViewBuilder.closeSidebar();
+
+		Form.addTextField(fieldPositionNumber = "1");
+
+		Form.gotoAutocompleteTab();
+
+		Form.enableAutocompleteConfiguration();
+
+		FormFields.selectRadioOption(
+			fieldName = "",
+			radioOption = "Manually");
+
+		FormFields.editOptionsRow(
+			optionName = "Option1",
+			optionValue = "Option 1",
+			rowNumber = "1");
+
+		FormFields.editOptionsRow(
+			optionName = "Option2",
+			optionValue = "Option 2",
+			rowNumber = "2");
+
+		Form.saveElementSet();
+
+		FormsAdminNavigator.gotoBack();
+
+		Navigator.gotoNavItem(navItem = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddElementSet(
+			elementSetTitle = "Element Set 1",
+			fieldPositionNumber = "0");
+
+		FormsAdminNavigator.gotoEditField(fieldName = "Text");
+
+		Form.gotoAutocompleteTab();
+
```

Comment:
Hi @ewertonluna! According to the [ticket](https://issues.liferay.com/browse/LPS-133464), the following step is missing. Please, could you confirm it and add it? Thanks!! =)

> 13. Assert the "Autocomplete" switch is toggled.